CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## NEXT RELEASE
+
+- Fixes multiple bugs that made the `addCreditCard` function unusable, adds tests
```

Comment:
I'm not seeing any diff lines here re: changing this function. Can you point me to them?